factory.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
```

Comment:
Your Factory pattern should have `BicycleFactory` with `CreateFourWellBicycle` and `CreateTwoWheelBicycle` methods.

we can discuss it internally in the next class.